packages/eslint-config-wpcalypso/index.js

```diff
@@ -109,7 +109,30 @@ module.exports = {
 		// Assumed by default with Babel
 		strict: [ 2, 'never' ],
 		'template-curly-spacing': [ 2, 'always' ],
-		'valid-jsdoc': [ 2, { requireReturn: false } ],
+
+		// jsdoc plugin
+		'jsdoc/check-access': 2,
+		'jsdoc/check-alignment': 2,
+		'jsdoc/check-param-names': 2,
+		'jsdoc/check-tag-names': [ 2, { definedTags: [ 'format', 'jest-environment' ] } ],
```

Comment:
How about this:
- config package leaves this rule `"off"`
- calypso can opt in to the rule
- we allow `jest-environment` but _not_ `format`. That _should_ disappear in Calypso 🙂

Reply:
Pushed in b4ca840dee6b368ec696879bf9599bccd760ed3d